e2e/basic/suite_setup_test.go

```diff
@@ -0,0 +1,41 @@
+package basic
+
+import (
+	"context"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+type Suite struct {
+	suite.Suite
+	Knuu *knuu.Knuu
+}
+
+func (s *Suite) SetupSuite() {
+	var (
+		err error
+		ctx = context.Background()
+	)
+	s.Knuu, err = knuu.New(ctx)
+	s.Require().NoError(err)
+	s.FailNow("Error cleaning up test suite: %v", err)
+	s.Knuu.HandleStopSignal(ctx)
+}
+
+func (s *Suite) TearDownSuite() {
+	s.T().Cleanup(func() {
+		logrus.Info("Tearing down test suite...")
+		err := s.Knuu.CleanUp(context.Background())
+		if err != nil {
+			s.FailNow("Error cleaning up test suite: %v", err)
+		}
```

Comment:
You can simplify this:
```suggestion
		s.Require().NoError(s.Knuu.CleanUp(context.Background()))
```